semeio/workflows/correlated_observations_scaling/job_config.py

```diff
@@ -171,48 +180,119 @@ def _CALCULATE_KEYS_key_not_empty_list(content):
 }
 
 
-def build_schema():
-    return {
-        MK.Type: types.NamedDict,
-        MK.Description: "Keys and index lists from all scaled keys",
-        MK.LayerTransformation: _expand_input,
-        MK.Content: {
-            "CALCULATE_KEYS": {
-                MK.Type: types.NamedDict,
-                MK.Description: _CALCULATE_KEYS_DESCRIPTION,
-                MK.Content: {
-                    "keys": _KEYS_SCHEMA,
-                    "threshold": {
-                        MK.Type: types.Number,
-                        MK.ElementValidators: (_min_max_value,),
-                        MK.Description: "Threshold used when computing primary"
-                        "components of the clusters.",
-                        MK.Default: 0.95,
-                    },
-                    "std_cutoff": {
-                        MK.Type: types.Number,
-                        MK.Description: "A lower bound on the ensemble standard"
-                        " deviation. All data points with insufficient variation"
-                        " will be dropped. The value is defaulted to the value used"
-                        " in the ert run, but if a value is given in the config, "
-                        " that value will be used",
-                    },
-                    "alpha": {
-                        MK.Type: types.Number,
-                        MK.Description: "Scalar controlling the allowed distance"
-                        "between ensemble mean and observation. In particular, "
-                        "if: `abs(observed_value - ensemble_mean) > "
-                        "alpha * (ensenmble_std + observed_std)` the data point "
-                        "will be dropped. The value is defaulted to the value used "
-                        "in the ert run, but if a value is given in the config, "
-                        "that value will be used",
-                    },
+_CORRELATED_OBSERVATIONS_SCHEMA = {
```

Comment:
No changes here, this is just formatting (and changed from function to variable.

Reply:
But was there a specific reason for doing this change? From function to variable?

Reply:
I saw that the author of `ConfigSuite` chose that style, so I conformed.

Reply:
Alright, I don't have any objections really. As long as we don't get a mix of one over the other between jobs now? I think there are more jobs in semeio that use `ConfigSuite`, and I think we should honour our own structure as well.

Reply:
I changed it to be internally consistent, because `misfit_preprocessor` is using dict.